### PR TITLE
fix: Css fixes/platform observations

### PR DESCRIPTION
**Pull request by mukultayal135**
# Description

This PR includes the UI changes suggested under New Misc and Platform observations
New Misc
1) Create app
2) Helm Apps list
3) App details > URLs drawer
4) App list
5) App overview > Project
6) App Details & Resource Browser
7) Command bar

Platform Observations

1) Indentation broken in release notes
2) Change View chart CTA
3) Stop content expansion
4) Automatic trimming of name
5) Metrics for usage in cluster

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- Reviewed By Utkarsh

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/ClusterNodes/ClusterOverview.tsx [hunk 1]
```diff
@@ -141,54 +163,81 @@ export default function ClusterOverview({
...
+    const tippyForMetricsApi = () => {
+        return (
+            <>
+                <span>NA</span>
+                <TippyCustomized
+                    theme={TippyTheme.white}
+                    className="w-300 h-100 fcv-5 dc__align-left"
+                    placement="bottom"
+                    Icon={QuestionFilled}
+                    heading={'Metrics API is not available'}
```
Comment (vivek-devtron) [inline src/components/ClusterNodes/ClusterOverview.tsx:175]: Please remove the interpolation

File: src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Logs.component.tsx [hunk 2]
```diff
@@ -384,84 +382,108 @@ function LogsComponent({
...
-                            </React.Fragment>
+                                </React.Fragment>
+                            </>
                         )}
 
                         {(podContainerOptions?.containerOptions ?? []).length > 0 && (
                             <React.Fragment>
+                                <div className="h-16 dc__border-right ml-8 mr-8"></div>
                                 <div className="cn-6 ml-8 mr-10">Container </div>
-                                <div style={{ width: '150px' }}>
```
Comment (vivek-devtron) [inline src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Logs.component.tsx:464]: is removal of this intentional?

Comment (mukultayal135) [reply src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Logs.component.tsx:464]: Yes, there was demand for content driven select's width

File: src/components/ClusterNodes/ClusterOverview.tsx (hunk 1 above)
Comment (mukultayal135) [reply src/components/ClusterNodes/ClusterOverview.tsx]: done

File: src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Logs.component.tsx (hunk 2 above)
Comment (mukultayal135) [reply src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Logs.component.tsx:464]: Added max width of 200px as decided

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**